packages/notifi-react-example/package.json

```diff
@@ -43,10 +43,10 @@
   },
   "scripts": {
     "start": "BROWSER=none react-app-rewired start",
-    "build": "react-app-rewired build",
```

Comment:
We do not need to build notifi-react-example. for the following performance benefits
- local build speed and potential error (warning)
- CI (Github action) faster and avoid potential error and warning.